modules/ocl/cl_kernel.h

```diff
@@ -132,6 +132,7 @@ class CLKernel {
     SmartPtr<CLKernel>    _parent_kernel;
     CLArgList             _arg_list;
     CLWorkSize            _work_size;
+    char                  _kernel_cache_path[XCAM_MAX_STR_SIZE];
```

Comment:
_kernel_cache_path is for all kernels directory. so should be static member variables and protected by _kernel_map_mutex